components/drillDescription.js

```diff
@@ -28,10 +30,40 @@ export default function DrillScreen(props) {
 
   const windowWidth = Dimensions.get("window").width;
 
+  const sortedInputs = drillInfo.inputs.sort((a, b) => a.id > b.id);
```

Comment:
Maybe sort this by prettyTitle so when prettyTitle starts with a different letter as the id, it's still sorted correctly. Maybe map it to prettytitle before sorting would be good too.